roles/libvirt_manager/tasks/deploy_layout.yml

```diff
@@ -162,7 +162,6 @@
   ansible.builtin.copy:
     dest: "{{ cifmw_libvirt_manager_basedir }}/artifacts/interfaces-info.yml"
     content: "{{ cifmw_libvirt_manager_mac_map | to_nice_yaml }}"
-    force: false
 
 - name: Ensure we get proper access to CRC
   when:
```

Comment:
ack (defaults true had to check ;)) https://docs.ansible.com/ansible/latest/collections/ansible/builtin/copy_module.html#parameter-force